src/components/sdk/MeetingControls/AudioInputControl.tsx

```diff
@@ -39,7 +45,7 @@ const AudioInputControl: React.FC<Props> = ({
 
   return (
     <ControlBarButton
-      icon={<Microphone muted={muted} />}
+      icon={<Microphone muted={muted} mutedTitle={mutedIconTitle} unmutedTitle={unmutedIconTitle} />}
```

Comment:
Just checking if we do not pass any prop value to these titles here from `AudioInputControl` then are the values in the `Microphone` component `undefined` overriding the default ones? Or should we have default values for the `mutedIconTitle` and `unmutedIconTitle` in the `AudioInputControl`?

Reply:
If we don't pass any prop values to the `AudioInputControl`, the default ones on `Microphone` work ok. They don't get overridden by the `undefined` ones.

We decided to keep the defaults only at one level, because the default values are very much tied to the fact that it's a Microphone. However, if you'd prefer that we had some defaults set in the `AudioInputControl` as well, I can add them in. What do you think?

Reply:
Thanks for clarification, if it is not getting override and taking the icon title default then we should be fine.